receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -107,15 +142,42 @@ metrics:
       value_type: int
     attributes: [cache_name]
     enabled: true
-  elasticsearch.node.fs.disk.available:
```

Comment:
Since this was pre-existing, this would be a breaking change. Again, I didn't realize my suggestion would affect any existing metrics. Let's restore this name and, if it makes sense, use the namespace for related metrics.